engine/flink/engine/src/main/scala/pl/touk/nussknacker/engine/process/compiler/FlinkProcessCompiler.scala

```diff
@@ -65,10 +63,8 @@ class FlinkProcessCompiler(creator: ProcessConfigCreator,
     val compiledProcess =
       ProcessCompilerData.prepare(process, definitions(processObjectDependencies), listenersToUse, userCodeClassLoader, resultCollector, runMode)
 
-    val compiledExceptionHandler = validateOrFailProcessCompilation(compiledProcess.compileExceptionHandler())
-    val listeningExceptionHandler = new ListeningExceptionHandler(listenersToUse,
-      //FIXME: remove casting...
-      compiledExceptionHandler.asInstanceOf[FlinkEspExceptionHandler])
+    val configurableExceptionHandler = new ConfigurableExceptionHandler(process.metaData, processObjectDependencies, userCodeClassLoader)
+    val listeningExceptionHandler = new ListeningExceptionHandler(listenersToUse, configurableExceptionHandler)
```

Comment:
Let's use the runMode here to detect running in test and make `FlinkTestMain.ignore real exception handler` happy :)

Reply:
Also: remove   `StubbedFlinkProcessCompiler.prepareExceptionHandler`

Reply:
Removed prepExcHanlder from StubbedCompiler:
https://github.com/TouK/nussknacker/pull/2484/commits/15445eea3ccc8413766d0216cde2a08876504dcb

RunMode.Test uses empty handler:
https://github.com/TouK/nussknacker/pull/2484/commits/d7911e8c0b3158e3e41a4d84c1c4223d0d87045c